### DISCUSSION TITLE
I Think We Need A CRT App Just Like NTSC-RS

### DISCUSSION THREAD

**Discussion by JaegerAxelRetro2000s** [ntsc-rs/ntsc-rs#11] [Ideas]
Please Get Her/Him Some Ideas Here:

Edit: I Made The Logo For It After Like.. One Year?
<img width="1053" height="256" alt="image" src="https://github.com/user-attachments/assets/a20c9cee-e9b5-48f5-b78b-3257c75cf431" />

**Comment by Degamisu** [comment]
what do you mean CRT app? a plugin for retro game emulators?

**Reply by TismoDito** [reply]
He means an app that emulates the "camera pointing at CRT tv" trope that occurs in some videopastas.

**Reply by Degamisu** [reply]
simple. 2 ways, either do it IRL, or make a TV in blender and render it there.

**Reply by JaegerAxelRetro2000s** [reply]
You Know.. That Will Be A Good Idea!

**Reply by smfthe1199** [reply]
But the problem is, I don't have a CRT TV Beforehand, and also, I got a CRT TV but my second problem is that the image isn't full. The third problem is that it requires other items such as HDMI to AV converter, and need something to display. Eccentric option I recommend is to VHS-ify the video using a real VCR and real VCR Tape, or to make a custom analog transmitter, which requires MORE tools than ever. The cheaper option? Is to VHS-ify your video, using Audacity, and using ntsc-rs and a Video editor you may or may not have to make a VHS video ready to put to a CRT Set before recording using camera.

**Reply by Degamisu** [reply]
when i "vhsify" my videos/images, i use that HDMI to RCA converter. Even though you don't have it, i recommend to get it. Its like 15 bucks on ebay or something. Its very worth it though.

**Reply by smfthe1199** [reply]
Yes. I acknowledge it. Another suggestion is to use a DVD Player. Put my video into the VHS Player or into the internal CRT Set to VHS-ify, or CRT-ify my videos.

**Reply by smfthe1199** [reply]
A far more cheap but also time-consuming solution is to get a CRT effect in Alight Motion.

**That's Right**, and you can do it.

**Reply by smfthe1199** [reply]
Here: https://drive.google.com/file/d/1JJvn6wRWP3Y4SBuEMxkLHlToJ3lzjyyq/view?usp=drivesdk

**Reply by Degamisu** [reply]
Cool! i think the fisheye was a little much, most of them and including mine have only a slight curve, also the scanlines arent _too_ visible irl. And one more suggestion, try (if possible of course) to have phosphor trails. this is the trails you see on a dark background against a bright moving sprite. it occurs when the excited TV material Phosphors is fading out of its light faze!

other than those 3, good job!

**Reply by smfthe1199** [reply]
I already did the "Phosphor" thing and the results was astonishing, speaking of the scanlines, I'll probably lower the opacity of this.

**Reply by smfthe1199** [reply]
Like this:
![Screenshot_2024-05-24-02-13-52-85_da12b4d69ba00bd9279f440202b88de5](https://github.com/valadaptive/ntsc-rs/assets/153495179/84070790-65b3-415e-a1f4-34169258a6cf)

**Reply by Degamisu** [reply]
i like it! maybe even a little less fisheye. then it should be perfect. (oh you can also increase the brightness or clipping too to get it even better)

**Reply by smfthe1199** [reply]
Noting that this effect wasn't originally made by mine, it was made by someone whose current YouTube Name and Handle is Palawan Journos (The handle is @palawanjournos01official), but at least I downloaded the video before it was lost in time.

**Reply by smfthe1199** [reply]
Here, Even perfected:
![Screenshot_2024-05-24-02-22-59-98_da12b4d69ba00bd9279f440202b88de5](https://github.com/valadaptive/ntsc-rs/assets/153495179/97ebef63-6548-4fab-928b-6e412dd16528)

**Reply by Degamisu** [reply]
thats perfect! great job.

**Reply by smfthe1199** [reply]
Here is another video I made with is a CRT effect with scanlines and phosphor, and inside a VHS (effect by True VHS) with a Date on the bottom left: https://drive.google.com/file/d/1JbbSGTAXYUbDXEzML6JM9yr0jY6_fwTz/view?usp=drivesdk

**Reply by smfthe1199** [reply]
I think I obtained the aforementioned user in my reply and he was also known as D3X.

**Reply by Degamisu** [reply]
cool! i wouldnt recommend the rolling shutter screen, as it is usually only the phosphor decay that is in there, which is hard to replicate in video.

**Reply by smfthe1199** [reply]
> cool! i wouldnt recommend the rolling shutter screen, as it is usually only the phosphor decay that is in there, which is hard to replicate in video.

Noting that CRT monitors are like this as you saw, but different because it's replicated.

**Reply by JaegerAxelRetro2000s** [reply]
> what do you mean CRT app? a plugin for retro game emulators?

Well I Ment It For Adding MP4 Files

**Comment by fuki98** [comment]
Here's a very good emulations of video signals. https://github.com/LMP88959/NTSC-CRT
If you mean a real CRT, do it yourself.

**Reply by smfthe1199** [reply]
I am VERY interested in trying this and I am looking forward to try to get it to my PC.

**Comment by Maris495** [comment]
ShaderGlass could be useful to overlay a video or a game with a CRT shader and such.
https://github.com/mausimus/ShaderGlass

**Reply by smfthe1199** [reply]
Great.

**Comment by zumzum360** [comment]
like a CRT-RS?

**Reply by JaegerAxelRetro2000s** [reply]
Yep

**Reply by mariusbigelis-cyber** [reply]
I have some crt tv textures for this 
<img width="1600" height="1200" alt="1570133726433-scanlines-50-no-gray-frame (1)" src="https://github.com/user-attachments/assets/0cad9bb0-ded6-48db-b182-0b3d36cd4aee" />
<img width="1600" height="1200" alt="1570133726433-scanlines-50-no-gray-frame" src="https://github.com/user-attachments/assets/0550556a-93e2-4bec-9ead-21226fe9ae53" />
<img width="1600" height="1200" alt="1570133708809-scanlines-50-1600x1200" src="https://github.com/user-attachments/assets/79bc93a3-2c22-44e2-b9fe-370043e14c91" />
<img width="1024" height="768" alt="1570056759449-crt-border-scanlines-50-1024x768" src="https://github.com/user-attachments/assets/b68e6227-98c0-4942-97a3-bcec227e6e8a" />
<img width="1750" height="1344" alt="1992698e0c5232f6300b7534482a92f51282fca7" src="https://github.com/user-attachments/assets/7c70f3c9-68e5-48da-b928-fb2a16f02fc2" />

**Reply by mariusbigelis-cyber** [reply]
Those blank crt photos are png textures of crt tv with scanlines

**Reply by JaegerAxelRetro2000s** [reply]
Oh Cool!

**Comment by JaegerAxelRetro2000s** [comment]
Yo @mariusbigelis-cyber I Made The Logo For CRT-RS
<img width="1053" height="256" alt="image" src="https://github.com/user-attachments/assets/15a42a2c-75ea-4787-92da-deb33a45c54d" />

**Reply by mariusbigelis-cyber** [reply]
That's so cool!

**Reply by mariusbigelis-cyber** [reply]
Guys i made some good kind of vhs preset ntsc rs
<img width="1080" height="789" alt="Screenshot_20260914_211444_CapCut" src="https://github.com/user-attachments/assets/86b53c5c-3864-4b2f-ae32-070b4e4a4266" />
<img width="1080" height="832" alt="Screenshot_20260913_140437_CapCut" src="https://github.com/user-attachments/assets/837cfe0c-ea0f-4edd-8da8-8f52269f3c1a" />

[preset-23.json](https://github.com/user-attachments/files/32229153/preset-23.json)

Also go to capcut add dreamy halo effect to photo or video and turn off dreamy halo settings leaving only blur setting in 14% dont for get to put size in ntsc rs from 500 lines to 630 lines

**Reply by JaegerAxelRetro2000s** [reply]
That's Cool!

**Reply by mariusbigelis-cyber** [reply]
I'm been thinking about now how you gonna make app crt-rs like you gonna make it using app termux or something to code or not.
